DogServiceExample/src/test/java/war/DogTest.java

```diff
@@ -1,63 +0,0 @@
-package war;
-
-import java.io.BufferedInputStream;
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.util.Calendar;
-import java.util.Date;
-import java.text.DateFormat;
-import java.text.SimpleDateFormat;
-import junit.framework.Assert;
-import org.junit.Test;
-import app.Dog;
-
-public class DogTest {
-    DateFormat simpleDateFormat = new SimpleDateFormat("dd.MM.yyyy");
-    Calendar c = Calendar.getInstance();
-    String dateToSubmit;
-
-    @Test
-    public void setNameForDog() {
-        Dog dog1 = new Dog();
-        String tempName = dog1.getName();
-        dog1.setName(""); //try to set empty
-        Assert.assertTrue(tempName == dog1.getName());
-
-        dog1.setName("12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901"); //try to set >101 symbol
-        Assert.assertTrue(tempName == dog1.getName());
-
-        dog1.setName("q"); //try to enter min value
-        Assert.assertTrue(dog1.getName() == "q");
-
-        dog1.setName("1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890"); //try to enter max value
-        Assert.assertTrue(dog1.getName() == "1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890");
-    }
-
-    @Test
-    public void setDateOfBirthForDog() {
```

Comment:
Tests check too much stuff.
Split into several tests: one for empty, one for future date, etc.